routes/api/resources.js

```diff
@@ -50,11 +64,24 @@ router.get('/:id', async (req, res) => {
   }
 });
 
-router.patch('/:id', async (req, res) => {
+router.patch('/:id', interceptors.requireLogin, async (req, res) => {
```

Comment:
I've added the requireLogin interceptor to both the POST and PATCH routes (create and update), otherwise ANYBODY could send a request without an account and create a Resource in the system...!